### PR TITLE
Embedded resources may not have extension in the name

### PR DESCRIPTION
**Pull request by dreddy-work**
Winforms run-time is replacing the embedded bitmap resources to Icons that match with VisualStudio styles. Winforms also made change to logical names ( earlier they contain file name with extension ".bmp". now that the files are changed ti .ICO files, we are removing the extension from logical names for future proof)

### CODE REVIEW

**Comment by dreddy-work** [comment]
Corresponding Winforms run-time changes are in [this ](https://github.com/dotnet/winforms/pull/678) PR.

**Comment by danmosemsft** [comment]
Is this for Preview 5? With the fork in Monday, if there is a matching winforms change, we should make sure both are in or both are out.

File: src/System.Drawing.Common/src/System/Drawing/ToolboxBitmapAttribute.cs [hunk 1]
```diff
@@ -282,6 +282,10 @@ internal static Image GetImageFromResource(Type t, string imageName, bool large,
                     {
                         name = name.Substring(indexDot + 1);
                     }
+
+                    // All bitmap images from winforms runtime are changed to Icons
+                    // and logical names, now, does not contain any extension.
+                    rawbmpname = name;
```
Comment (danmosemsft) [inline src/System.Drawing.Common/src/System/Drawing/ToolboxBitmapAttribute.cs]: Is a test possible? We do have tests for this class.

**Comment by dreddy-work** [comment]
This is a less risky change on corefx side and independant of Winforms change (if winforms change can't make it, it will be no-op on corefx).  Modernizing winforms icons was Preview #5 item. I can look into a possible test.

**Comment by dreddy-work** [comment]
@danmosemsft, are you suggesting we hold this until Preview snap is done and go to Preview 6?

**Comment by danmosemsft** [comment]
@dreddy-work it's up to you. If it's not needed for preview 5 I suggest to wait a few hours. If it is required, feel free to merge.

**Comment by dreddy-work** [comment]
@danmosemsft , @safern  can you please help merge this?

**Comment by danmosemsft** [comment]
done.

**Pull request merged**